external/mqtt/src/main/scala/org/apache/spark/streaming/mqtt/MQTTInputDStream.scala

```diff
@@ -55,14 +55,14 @@ class MQTTInputDStream(
     brokerUrl: String,
     topic: String,
     storageLevel: StorageLevel
-  ) extends ReceiverInputDStream[String](ssc_) with Logging {
```

Comment:
No, it was not necessary, I just noticed that it wasn't needed, so I thought I should be a boy scout :)